project/frontend/src/components/SGonksLists/LongSGonksList/LongSGonkRow/LongSGonkRow.js

```diff
@@ -19,7 +43,9 @@ const LongSGonkRow = (props) => {
         {priceDelta >= 0 ? "+" : "-"}t${Math.abs(priceDelta)}
```

Comment:
priceDelta represents the amount earned / lost right? Did we not decide to calculate this at the backend? Can you clarify why it's needed here?